notebooks/document-chunking/with-index-pipelines.ipynb

```diff
@@ -0,0 +1,561 @@
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "f38e0397",
+    "outputId": "ad6df489-d242-4229-a42a-39c5ca19d124"
+   },
+   "outputs": [],
+   "source": [
+    "from elasticsearch import Elasticsearch\n",
+    "from getpass import getpass\n",
+    "\n",
+    "# https://www.elastic.co/search-labs/tutorials/install-elasticsearch/elastic-cloud#finding-your-cloud-id\n",
+    "ELASTIC_CLOUD_ID = getpass(\"Elastic Cloud ID: \")\n",
+    "\n",
+    "# https://www.elastic.co/search-labs/tutorials/install-elasticsearch/elastic-cloud#creating-an-api-key\n",
+    "ELASTIC_API_KEY = getpass(\"Elastic Api Key: \")\n",
+    "\n",
+    "# Create the client instance\n",
+    "client = Elasticsearch(\n",
+    "    # For local development\n",
+    "    # hosts=[\"http://localhost:9200\"] \n",
+    "    cloud_id=ELASTIC_CLOUD_ID,\n",
+    "    api_key=ELASTIC_API_KEY,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fcd165fa",
+   "metadata": {
+    "id": "fcd165fa"
+   },
+   "source": [
+    "If you're running Elasticsearch locally or self-managed, you can pass in the Elasticsearch host instead. [Read more](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/connecting.html#_verifying_https_with_certificate_fingerprints_python_3_10_or_later) on how to connect to Elasticsearch locally."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1462ebd8",
+   "metadata": {
+    "id": "1462ebd8"
+   },
+   "source": [
+    "Confirm that the client has connected with this test."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "25c618eb",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "25c618eb",
+    "outputId": "30a6ba5b-5109-4457-ddfe-5633a077ca9b"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'name': 'instance-0000000011', 'cluster_name': 'd1bd36862ce54c7b903e2aacd4cd7f0a', 'cluster_uuid': 'tIkh0X_UQKmMFQKSfUw-VQ', 'version': {'number': '8.11.1', 'build_flavor': 'default', 'build_type': 'docker', 'build_hash': '6f9ff581fbcde658e6f69d6ce03050f060d1fd0c', 'build_date': '2023-11-11T10:05:59.421038163Z', 'build_snapshot': False, 'lucene_version': '9.8.0', 'minimum_wire_compatibility_version': '7.17.0', 'minimum_index_compatibility_version': '7.0.0'}, 'tagline': 'You Know, for Search'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(client.info())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e272c75",
+   "metadata": {},
+   "source": [
+    "## Load Model from hugging face\n",
+    "The first thing you will need is a model to create the text embeddings out of the chunks, you can use whatever you would like, but this example will run end to end on the all-distilroberta-v1 model. With an Elastic Cloud cluster created or another Elasticsearch cluster ready, we can upload the text embedding model using the eland library."
```

Comment:
Question: In the example below we use `sentence-transformers__all-minilm-l6-v2` but the description mentions using `all-distilroberta-v1` model.  Are both of these models same? 

Is see same in the article - https://www.elastic.co/search-labs/blog/articles/chunking-via-ingest-pipelines

Reply:
thanks for spot! Yeh it uses the minilm-l6-v2 text embedding model. Yep this notebook is the article eq. its for those who want to apply this pipeline from within a notebook.